src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/AddMyExtensionsDialog.fr.xlf

```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="colHeaderExtensionName.Text">
         <source>Name</source>
-        <target state="translated">Nom</target>
+        <target state="translated">Nom </target>
```

Comment:
This whitespace looks suspect. Extra spaces might cause layout problems.

Is there ever a case where we wouldn't want translated strings to be trimmed?